proxies/java/src/main/java/com/dvc_harness/proxy/controller/ProxyController.java

```diff
@@ -28,6 +30,36 @@ public Spec spec() {
         return new Spec();
     }
 
+    @PostMapping("/user")
+    public BaseResponse user(@RequestBody UserRequestBody body, HttpServletResponse response) {
+        try {
+            User sdkUser = User.builder()
+                .userId(body.user_id)
+                .name(body.name)
+                .language(body.language)
+                .country(body.country)
+                .appVersion(body.appVersion)
+                .appBuild(body.appBuild)
+                .customData(body.customData)
+                .privateCustomData(body.privateCustomData)
+                .createdDate(body.createdDate)
+                .lastSeenDate(body.lastSeenDate)
+                .build();
+
+            var userId = DataStore.Users.size();
+            DataStore.Users.put(userId + "", sdkUser);
```

Comment:
Might be more clear to do `var userId = Interger.toString(DataStore.Users.size());`

Reply:
👍